build.gradle.kts

```diff
@@ -12,9 +12,21 @@ configurations.all {
 
 plugins {
     kotlin(Plugins.Kotlin.PLUGIN_JVM) version Versions.KOTLIN
+    id(Plugins.DETEKT_PLUGIN) version Versions.DETEKT
     id(Plugins.BEN_MANES_PLUGIN) version Versions.BEN_MANES
 }
 
+tasks.named("detekt") {
+    dependsOn(":test_runner:detekt")
+    dependsOn(":integration_tests:detekt")
+    dependsOn(":flank-scripts:detekt")
+}
+
+tasks.named("clean"){
```

Comment:
Is it necessary? This task already invokes those:

![Screenshot 2020-09-30 at 09 09 18](https://user-images.githubusercontent.com/32893017/94654141-ef049080-02fc-11eb-8f9e-85de2a5a7600.png)

Or is it the order that matters?

Reply:
the clean was for testing will remove